### DISCUSSION TITLE
Star for star is getting out of hand

### DISCUSSION THREAD

**Discussion by sid77x** [community/community#21070] [Other Feature Feedback, Questions, & Ideas] [answered]
After the addition of the new "Starstruck" achievement badge in github, people have started to ask for star for star where you basically star their reps and they star yours, thus stars have lost all their meaning. It makes getting stars easy and its not about the merit or the quality of the repo anymore. I don't want to be THAT guy but honestly i hope github does something about this

**Comment by MrSarno** [comment, marked as answer]
Eh. While it does devalue the badge somewhat, I don't think GitHub want to get into the weeds of deciding when one has truly been earned, and when it hasn't. Should someone have their Galaxy Brain badge revoked / down-leveled if there is a consensus that one or more of the discussions they answered was asking a pretty obvious question? The slope can be kind of slippery.

Personally, I view badges as callouts that bring your attention to things you can then track down yourself. If someone had the Starstruck badge I would be inclined to look through their repositories to see which had been starred so much and what it was about. If it was clear that so many people were unlikely to hold a genuine interest in such a repo, that would contribute to my opinion of that person.

I guess the tl;dr is that, in the eyes of anyone doing their homework, that effectively becomes a badge of shame.

**Comment by radar-blip** [comment]
I heard something somewhere once, something like: "when you cheat, you're only cheating yourself". I think it was Yoda.

**Reply by sid77x** [reply]
true lol
